src/main/java/teammates/ui/controller/AdminAccountDetailsPageData.java

```diff
@@ -7,25 +8,68 @@
 import teammates.common.datatransfer.CourseDetailsBundle;
 import teammates.common.util.Const;
 import teammates.common.util.Url;
+import teammates.ui.template.AdminAccountDetailsInstructorCourseListTableRow;
+import teammates.ui.template.AdminAccountDetailsStudentCourseListTableRow;
 
 public class AdminAccountDetailsPageData extends PageData {
     
-    public AccountAttributes accountInformation;
-    public List<CourseDetailsBundle> instructorCourseList;
-    public List<CourseAttributes> studentCourseList;
-
-    public AdminAccountDetailsPageData(AccountAttributes account) {
+    private AccountAttributes accountInformation;
+    private List<AdminAccountDetailsInstructorCourseListTableRow> instructorCourseListTable;
+    private List<AdminAccountDetailsStudentCourseListTableRow> studentCourseListTable;
+    
+    public AdminAccountDetailsPageData(AccountAttributes account, AccountAttributes accountInformation, 
+                                       List<CourseDetailsBundle> instructorCourseList, 
+                                       List<CourseAttributes> studentCourseList) {
         super(account);
+        this.accountInformation = accountInformation;
+        this.instructorCourseListTable = createInstructorCourseListTable(instructorCourseList);
+        this.studentCourseListTable = createStudentCourseListTable(studentCourseList);
+    }
+    
+    private List<AdminAccountDetailsStudentCourseListTableRow> createStudentCourseListTable(
+                                    List<CourseAttributes> studentCourseList) {
+        List<AdminAccountDetailsStudentCourseListTableRow> courseListTable = new ArrayList<AdminAccountDetailsStudentCourseListTableRow>();
+        if (studentCourseList != null) {
+            for(CourseAttributes courseDetails : studentCourseList) {
+                AdminAccountDetailsStudentCourseListTableRow row = new AdminAccountDetailsStudentCourseListTableRow(accountInformation.googleId, courseDetails);
+                courseListTable.add(row);
+            }
+        }
+        return courseListTable;
+    }
+
+    private List<AdminAccountDetailsInstructorCourseListTableRow> createInstructorCourseListTable(
+                                                            List<CourseDetailsBundle> instructorCourseList) {
+        List<AdminAccountDetailsInstructorCourseListTableRow> courseListTable = new ArrayList<AdminAccountDetailsInstructorCourseListTableRow>();
+        if (instructorCourseList != null) {
+            for(CourseDetailsBundle courseDetails : instructorCourseList) {
+                AdminAccountDetailsInstructorCourseListTableRow row = new AdminAccountDetailsInstructorCourseListTableRow(accountInformation.googleId, courseDetails);
+                courseListTable.add(row);
+            }
+        }
+        return courseListTable;
```

Comment:
Place a blank line before the return statement

if () {
}

return ...

Also for line 38, 69, 76